CHANGELOG.md

```diff
@@ -14,12 +14,14 @@ Here is a template for new release sections
 - measurement device (#1215)
 - parameterisation, model calibration (#1216)
 - scenario projection (#1217)
+- (electricity) import/export (value) (#1221)
```

Comment:
I think that listing the individual changes is more understandable to the reader.
Your style is clever, but at least for me a little ambiguous. 
Having a "dumb" list also enables automatic parsing of the `CHANGELOG.md` for power-users ;)

I added missing concepts (`sirup`, `electrical energy amount value`)

```suggestion
- import, export, electricity import, electricity export, electrical energy amount value, electricity import value, electricity export value, sirup (#1221)
```